src/utils/utils.js

```diff
@@ -220,8 +221,8 @@ function findAllEnabledDestinations(sdkSuppliedIntegrations, configPlaneEnabledI
     if (!allValue) {
```

Comment:
Please revert all the changes in this function

Reply:
changes reverted